system_tests/espresso_e2e_test.go

```diff
@@ -413,17 +214,17 @@ func TestEspressoE2E(t *testing.T) {
 	// Remember the number of messages
 	var msgCnt arbutil.MessageIndex
 	err = waitFor(t, ctx, func() bool {
-		cnt, err := node.ConsensusNode.TxStreamer.GetMessageCount()
+		cnt, err := l2Node.ConsensusNode.TxStreamer.GetMessageCount()
 		Require(t, err)
 		msgCnt = cnt
 		log.Info("waiting for message count", "cnt", msgCnt)
-		return msgCnt > 6
+		return msgCnt >= 2
```

Comment:
"msgCnt > 6" means there are probably 2 batches already. We used to have some bugs here. Although we should add some comments at least, or change another way to check the batch number.

Reply:
Ok. I forgot we are using the sovereign sequencer.